ui.py

```
@@ -3428,11 +3482,171 @@ def _refresh_queue_mode_label(self) -> None:
+                # sane cap). The 10s cache keeps this off the hot path.
+                start = max(0, size - 50_000_000)
+                with open(path, "r", encoding="utf-8", errors="ignore") as f:
+                    f.seek(start)
+                    tail = f.read()
+                matches = re.findall(
+                    r'"authenticateResponse"\s*:\s*\{[^}]*?"screenName"\s*:\s*"([^"]+)"',
+                    tail,
+                )
+                if matches:
+                    value = self._resolve_screenname_to_alias(str(matches[-1]).strip())
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)